package.json

```diff
@@ -13,6 +13,7 @@
     "build": "rimraf ./build && tsc && npm run copy-files",
     "copy-files": "ts-node copyNonTSFiles.ts",
     "json-server": "json-server ./src/services/mock-data.json --port 3300",
+    "classifier-json-server": "json-server --watch ./src/services/classifier-mock-data.json --port 3301",
```

Comment:
In an earlier PR, the micro service was integrated and we removed this.Why do we need to bring it back?

Reply:
Removed from package